src/mlpack/bindings/R/mlpack/src/Makevars

```diff
@@ -0,0 +1,3 @@
+PKG_CXXFLAGS = -DBOOST_MATH_PROMOTE_DOUBLE_POLICY=false -I. $(SHLIB_OPENMP_CXXFLAGS)
```

Comment:
I think we can directly use `$(SHLIB_OPENMP_CXXFLAGS)` without evoking the wraith of CRAN.

We may need to revisit this during the CRAN submission and add in a `Makevars.in` and `configure{.ac}` to determine if we should substitute in `@OPENMP_FLAG@`. 

https://github.com/RcppCore/RcppArmadillo/blob/eb0687bb6feb672151b1c1a42d79625a2ee027bb/src/Makevars.in#L3-L4